ApplicationLibCode/UserInterface/RiuResultInfoPanel.cpp

```diff
@@ -43,6 +43,12 @@ RiuResultInfoPanel::RiuResultInfoPanel( QWidget* parent )
     QVBoxLayout* layout = new QVBoxLayout();
     layout->addWidget( m_textEdit );
 
+    // Use a nonexisting font family to trigger the use of QFont::Monospace
+    // https://forum.qt.io/topic/35999/solved-qplaintextedit-how-to-change-the-font-to-be-monospaced/7
+    QFont font( "does not exist" );
+    font.setStyleHint( QFont::Monospace );
+    m_textEdit->setFont( font );
```

Comment:
Strange solution from 2013. Still the best way?

Reply:
Currently only way to get it working on Windows.